doc/quick_start_guide.md

```diff
@@ -13,14 +13,16 @@ Cygnus uses the subscription/notification feature of Orion. A subscription is ma
 ###Which storages is it able to integrate?
 Current stable release is able to persist Orion context data in:
 
-* HDFS, the Hadoop distributed file system.
-* MySQL, the well-know relational database manager.
-* CKAN, an Open Data platform.
+* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
+* [MySQL](https://www.mysql.com/), the well-know relational database manager.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5f3cda70057fb35842696996e986ba62f4a72e90